1.14.4/x86_64-bionic/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ubuntu:bionic
 
 # To improve : static hash make dynamic build of versions impossible.
 ARG VERSION=1.14.4
-ARG VERSION_HASH=6266235abe4bcbd41ea57bdf42f11ef89aa69f0386e8c8846d5228af69e7fa13
+ARG TARGETPLATFORM
```

Comment:
Can we give this a default value of `linux/amd64`? Otherwise specifying this becomes a requirement for building.

Reply:
If we set this value, the build won't be portable. That's what I commented in the PR comment. `TARGETPLATFORM` is appropriate to get a cleaner Dockerfile while using `buildx`. To have an image to build with `docker build` command, we may need to use `dpkg` to get the architecture.

Reply:
So buildx will not `--build-arg`, but instead do magic that cannot override a default value? Ouch! I'll do some more research, there must be a way to make it work with both.

Reply:
"When you invoke a build, you can set the --platform flag to specify the target platform for the build output, (for example, linux/amd64, linux/arm64, or darwin/amd64)."

This is from working with buildx.

Reply:
Can confirm passing --platform populates ARG TARGETPLATFORM and is accessible with $TARGETPLATFORM

Reply:
> So buildx will not --build-arg, but instead do magic that cannot override a default value? 

Idk exactly how it's done, but probably it's using --build-arg. If you don't use ARG keyword, TARGETPLATFORM isn't available. The issue is hard-coding an architecture as default value, when the image is will be build on a different architecture.

@xanimo I guess the best scenario would be to not force using `buildx`, and let users create the image using `docker build`.

Reply:
Let's leave this for now, I will propose a solution for doing simple `docker build .` in a separate PR.